main.js

```diff
@@ -0,0 +1,73 @@
+function growtree (clickEvent) {
+	// set object
+	var tree = new Object ();
```

Comment:
This object needs to be created elsewhere and passed into the growTree function, per the requirements